extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -438,7 +438,7 @@ export class AmpAnalytics extends AMP.BaseElement {
     request = this.expandTemplate_(request, trigger, event);
 
     // For consistency with amp-pixel we also expand any url replacements.
-    return urlReplacementsForDoc(this.win.document).expandAsync(request)
+    return urlReplacementsForDoc(this.getAmpDoc()).expandAsync(request)
```

Comment:
I am not completely sure if this is correct. I _think_ this will return the service for the main window. What we want is the service a) for the embed if inside an embed and b) for the top level doc if the element is not in embed.

Same above and below.